site/app/templates/grading/VersionChoice.twig

```diff
@@ -0,0 +1,22 @@
+<select style="margin: 0 10px;{{ formatting|raw }} " name="submission_version" onChange="{{ onChange|raw }}">
+
+    {% if gradeable.getActiveVersion() == 0 %}
+        <option value="0" {{ gradeable.getCurrentVersionNumber() == gradeable.getActiveVersion() ? "selected" : "" }}></option>
+    {% endif %}
+
+    {% for version in gradeable.getVersions() %}
+        <option value="{{ version.getVersion() }}" {{ version.getVersion() == gradeable.getCurrentVersionNumber() ? "selected" : "" }}>
+            Version #{{ version.getVersion() }}
+
+            {%- if gradeable.getNormalPoints() > 0 -%}
```

Comment:
Why are you using `{%- ... -%}` here instead of `{% ... %}` like in lines 3-5?

Reply:
`{%- ... -%}` is used to prevent extra spaces from appearing in the output.

Reply:
Ok, sounds cool